content/source/docs/enterprise/private/reliability-availability.html.md

```diff
@@ -226,30 +187,48 @@ The procedure here would be as follows:
   instance. The exact mechanism used is specific to the virtualization
   environment that the product runs in.
 
+**Restore**
+
 - Configure the virtualization to run the new snapshot and restart a new
   instance should it stop. In AWS, this is done using an Autoscaling Group.
   The environment can monitor the availability of port 443 for instance health
   as well.
```

Comment:
This is inaccurate.  The Replicated config contains references to the instance's IP address, which will change (even if an instance is stopped and then started).